README.md

```diff
@@ -54,5 +54,29 @@ will begin to drift slightly from the original design. This is expected and
 there is no expectation that the original design proposal needs to be updated
 to reflect these differences.
 
-The code and our documentation are the ultimate sources of truth. CFPs are merely
-the starting point for the implementation.
+The code and our documentation are the ultimate sources of truth. CFPs are
+the starting point for the implementation, but can also be updated as needed.
+
+# Status
```

Comment:
Might want to reference related projects: https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md

Reply:
If anything is relevant there, also ref: 

https://gateway-api.sigs.k8s.io/geps/overview/

along with rationale why the states exist.

Reply:
Good point on your first comment, I've opened an issue for it here https://github.com/cilium/design-cfps/issues/47